backdoor-ctf-2015/README.md

```diff
@@ -32,7 +32,7 @@ The contest problems will remain online forever on backdoor. You can chat with p
 
 ## External write-ups only
 
-* none yet
+* [binary/echo](https://shankaraman.wordpress.com/2015/04/02/backdoor-ctf-2015-echo-writeup/)
 
```

Comment:
no this is not the format that we use it here 

you just need to remove the write-up from missing and put it in external write-up only